src/style.css

```diff
@@ -1 +1,108 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+input {
+  width: 100%;
+  background: #fff;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  font-weight: 900;
+}
+
+.bars__big {
+  margin: 20px 0;
+}
+
+.big {
+  padding-left: 62px;
+  padding-bottom: 2px;
+  height: 70px;
+}
+
+.small {
+  padding-left: 33px;
+  padding-bottom: 5px;
+  height: 42px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.big__bar,
+.small__bar {
```

Comment:
in such cases, it's better to add a common class to both elements i.e. `bar`, and use it to add all styles that you need for big bar for example.

Then for the small bar, you should add the class `bar bar__small` and in `bar__small` overwrite all properties you need (`background-size` and `background-position` in this case)

Reply:
Fix it in all similar cases